integration/samples/material/baz/baz.component.scss

```diff
@@ -2,4 +2,5 @@
 
 .baz {
     color: $variable-defined-by-file-not-processed-by-loader;
+    content: "\2014 \00A0";
```

Comment:
Isn't that supposed to be valid unicode character in the css?

https://stackoverflow.com/a/10393517

Reply:
Ok, got it. The unicode character triggers `SyntaxError: Octal literal in strict mode`. Not so good....